### ISSUE TITLE
Include put, patch, delete into VTLResource

### ISSUE THREAD

**Issue by dsilvam** [dotCMS/core#15477]
This is already done - just for tracking

PR: https://github.com/dotCMS/core/pull/15469

**Comment by DeanGonzalez** [comment]
Just waiting on patch method for this one to pass QA

**Comment by DeanGonzalez** [comment]
needs qa on patch
